src/us/kbase/narrativemethodstore/db/mongo/test/MongoDynamicRepoDBTest.java

```diff
@@ -150,8 +150,8 @@ private void testRepo(boolean localFiles) throws Exception {
             }
             db.setRepoState(globalAdmin, repoModuleName, RepoState.testing);
             db.setRepoState(globalAdmin, repoModuleName, RepoState.disabled);
-            Assert.assertEquals(0, db.listRepoModuleNames(false, null).size());
-            Assert.assertEquals(1, db.listRepoModuleNames(true, null).size());
+            Assert.assertEquals(0, db.listRepoModuleNames().size());
+//            Assert.assertEquals(1, db.listRepoModuleNames(true, null).size());
```

Comment:
Why not just remove this?

Reply:
Sure, I can do that. I just kept them around since they're the only examples of calls to that method other than with (false, null).

I'll do it in a future PR though, as I already have a bunch of commits stacked on top of this one.

Are we good to merge otherwise?